### DISCUSSION TITLE
single dose and AUC

### DISCUSSION THREAD

**Discussion by WTY-tt** [Open-Systems-Pharmacology/Forum#1325] [Questions & Problems]
Hello, I recently started to use PK-SIM modeling to complete my graduation project. I found that the blood drug concentration curve of single dose simulated by software can only be simulated up to 24h, while many data in the literature are up to 72h. Will there be a big difference between the simulated AUC and that in the literature?
![单剂量文献-72h](https://user-images.githubusercontent.com/95362728/194460163-a977a38b-e141-47e4-9017-e3a94c260439.jpg)
![单剂量-软件-24h](https://user-images.githubusercontent.com/95362728/194460187-479510f7-f54e-478f-b435-86d66f52b70a.jpg)

**Comment by tobiasK2001** [comment]
Hi, you can adjust the simulation time in the setting of your simulation to 72h or any value you need as shown below:
![image](https://user-images.githubusercontent.com/25585248/194480777-76ab5409-715a-41c2-8712-031c65a3a773.png)

You might want to check out the comprehensive documentation here:
https://docs.open-systems-pharmacology.org/working-with-pk-sim/pk-sim-documentation/pk-sim-simulations#pk-parameters

Best, Tobias

**Reply by WTY-tt** [reply]
Thank you for your answer. I understand how to set it up.

**Comment by wilbertdew** [comment]
If your concern is that you have only data till 24h and data till 72 hours might change the AUC, a good check is to look at the difference between AUC 0-inf and AUC 0-tlast, with a simulation until 24h. This difference is also expressed in the parameter % AUC (tlast - infinity). The smaller this percentage, the lower the uncertainty in AUC induced by the limited sampling time. You can repeat this for a simulation till 72h if you really want to compare uncertainties between sampling time scenarios.

**Reply by WTY-tt** [reply]
Thank you for your reply. I will try as your reply.
